content/docs/command-reference/list-url.md

```diff
@@ -0,0 +1,97 @@
+# list-url
+
+> Aliased to `dvc ls-url`
+
+List contents from a supported URL (for example `s3://`, `ssh://`, and other
+protocols).
+
+> Useful to find data to `dvc get-url` or `dvc import-url`.
```

Comment:
Minor but ideally use proper admonitions for notes. I realize that this is copied from `list` (written before we had `<admon>`) and that it's @rlamy's first PR so here's what I mean:

```suggestion
<admon type="info">

Aliased to `dvc ls-url`

</admon>

List contents from a supported URL (for example `s3://`, `ssh://`, and other
protocols).

<admon type="tip">

Useful to find data to `dvc get-url` or `dvc import-url`.

</admon>
```

Related: https://github.com/iterative/dvc.org/issues/3382
More context: https://github.com/iterative/cml.dev/pull/210